IntegrationTests/IntegrationTests.xcodeproj/project.pbxproj

```diff
@@ -1270,6 +1270,7 @@
 				CLANG_ENABLE_MODULES = YES;
 				CODE_SIGN_STYLE = Automatic;
 				INFOPLIST_FILE = Runner/Info.plist;
+				IPHONEOS_DEPLOYMENT_TARGET = 12.0;
 				LD_RUNPATH_SEARCH_PATHS = (
 					"$(inherited)",
```

Comment:
blocking/ The `IPHONEOS_DEPLOYMENT_TARGET` should be managed through local [`Base.local.xcconfig`](https://github.com/DataDog/dd-sdk-ios/blob/f3b9a8f95a59fd955c772fa14273775d357840c5/Makefile#L36) which is git-ignored. Let's remove all changes to this file 🙏, otherwise this is dropping iOS 11 in CI.